ocp_resources/datavolume.py

```diff
@@ -223,13 +219,59 @@ def _check_none_pending_status(self, failure_timeout=120):
             for sample in TimeoutSampler(
                 wait_timeout=failure_timeout,
                 sleep=10,
-                func=lambda: self.instance.status.phase,
+                func=lambda: self.exists,
             ):
                 # If DV status is Pending (or Status is not yet updated) continue to wait, else exit the wait loop
-                if sample in [self.Status.PENDING, None]:
+                if sample and (
+                    not sample.status
```

Comment:
`not sample.status` I don't think this is needed, also it may be wrong.
We always have status anyway

Reply:
I failed on this :) it probably takes a few milliseconds,  but I ran it as is and in some of the runs (not all) I failed as the instance still didn't have any status